source/tutorial/install-mongodb-on-os-x.txt

```diff
@@ -29,17 +30,32 @@ information.
 Install MongoDB Community Edition
 ---------------------------------
 
-Prerequisite
-~~~~~~~~~~~~
+.. _osx-prereq:
+
+Prerequisites
+~~~~~~~~~~~~~
+
+If you have the Homebrew ``brew`` package installed on your OSX host 
+*and* you have previously tapped the official 
+`MongoDB Homebrew Tap <https://github.com/mongodb/homebrew-brew>`_, 
+skip to the :ref:`install` step.
```

Comment:
While this is great, I think our users in their haste may read this as skip the install step.

I'd go with skip the prerequisites and go to the ...